teamB/junhyungcho/app.js

```diff
@@ -1,33 +1,34 @@
+require('dotenv').config();
+
 const http = require('http');
 const express = require('express');
 const cors = require('cors');
 const morgan = require('morgan');
-const dotenv = require('dotenv');
-dotenv.config();
-
-const { DataSource } = require('typeorm');
-const myDataSource = new DataSource({
-    type: process.env.TYPEORM_CONNECTION,
-    host: process.env.TYPEORM_HOST,
-    port: process.env.TYPEORM_PORT,
-    username: process.env.TYPEORM_USERNAME,
-    password: process.env.TYPEORM_PASSWORD,
-    database: process.env.TYPEORM_DATABASE
-});
 
-myDataSource.initialize().then(() => console.log("Data Source has been initialized!"));
+const route = require('./routers');
 
 const app = express();
 
-app.use(express.json());
 app.use(cors());
-app.use(morgan('combined')); 
-app.get('/ping', cors(), (req, res, next) => res.json({ message : "pong" }));
+app.use(morgan('combined'));
+app.use(express.json());
+app.use(route);
+
+app.get('/ping', cors(), (req, res, next) => {
+    res.status(200).json({ message : "pong" })
+});
 
 const server = http.createServer(app);
-const PORT = process.env.PORT;
+const PORT = process.env.PORT || 3000;
+
 const start = async () => {
-    server.listen(PORT, () => console.log(`server is listening on ${PORT}`));
+    try {
+        server.listen(PORT, () => console.log(`Server is listening on ${PORT}`));
+    } catch (err) {
+        throw err;
+    } finally {
+        console.log('===========================================');
+    }
```

Comment:
1. throw을 사용하면 catch로 잡은 err를 보통은 해당 함수 보다 상위에 있는 catch에서 다시 잡게 됩니다. 그래서 지금 같은 상황에서는 그냥 바로 err를 잡고 console.log로 확인하는 정도면 충분 할 것 같습니다.

```suggestion
        server.listen(PORT, () => console.log(`Server is listening on ${PORT}`));
    } catch (err) {
        console.log(err);
    } 
```